de.ient.YUView.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: de.ient.YUView
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: YUView
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --env=QT_NO_FT_CACHE=1
+
+modules:
+  - name: YUView
+    buildsystem: qmake
+    sources:
+      - type: git
+        branch: deployment
```

Comment:
Since you seems to be an upstream contributor, it would be nice if you could add git tag / make a proper release. Flathub is meant be used for hosting stable releases of software.

Reply:
We're currently about to make a new stable release, but it's not done yet. Probably another week or two. While we're doing this we're also moving to flatpak. I'll set the tag, once we have the stable release.